app/feature/feature-claim-details/src/main/kotlin/com/hedvig/android/feature/claim/details/ui/ClaimDetailsDestination.kt

```diff
@@ -127,19 +155,123 @@ private fun ClaimDetailScreen(
+
+  val photoCaptureState = rememberPhotoCaptureState(appPackageId = appPackageId) { uri ->
+    logcat { "ChatFileState sending uri:$uri" }
+    onSendFile(uri)
+  }
+  val photoPicker = rememberLauncherForActivityResult(
+    contract = ActivityResultContracts.PickVisualMedia(),
+  ) { resultingUri: Uri? ->
+    if (resultingUri != null) {
+      onSendFile(resultingUri)
+    }
+  }
+  val filePicker = rememberLauncherForActivityResult(
+    contract = ActivityResultContracts.GetContent(),
+  ) { resultingUri: Uri? ->
+    if (resultingUri != null) {
+      onSendFile(resultingUri)
+    }
+  }
+
+  if (showFileTypeSelectBottomSheet) {
+    HedvigBottomSheet(
+      onDismissed = { showFileTypeSelectBottomSheet = false },
+      sheetState = sheetState,
+      content = {
+        Column {
+          TextButton(
+            onClick = {
+              photoPicker.launch(PickVisualMediaRequest())
+              showFileTypeSelectBottomSheet = false
+            },
+            modifier = modifier
+              .fillMaxWidth()
+              .padding(horizontal = 16.dp),
+            shape = MaterialTheme.shapes.squircleMedium,
+          ) {
+            Row(
+              horizontalArrangement = Arrangement.SpaceBetween,
+              modifier = Modifier
+                .fillMaxWidth()
+                .padding(vertical = 8.dp),
+            ) {
+              ProvideTextStyle(MaterialTheme.typography.bodyLarge) {
+                Text(text = stringResource(id = R.string.file_upload_photo_library))
+              }
+              Icon(imageVector = HedvigIcons.Pictures, contentDescription = "library icon")
+            }
+          }
+          Spacer(modifier = Modifier.height(4.dp))
+          TextButton(
+            onClick = {
+              photoCaptureState.launchTakePhotoRequest()
+              showFileTypeSelectBottomSheet = false
+            },
+            modifier = modifier
+              .fillMaxWidth()
+              .padding(horizontal = 16.dp),
+            shape = MaterialTheme.shapes.squircleMedium,
+          ) {
+            Row(
+              horizontalArrangement = Arrangement.SpaceBetween,
+              modifier = Modifier
+                .fillMaxWidth()
+                .padding(vertical = 8.dp),
+            ) {
+              ProvideTextStyle(MaterialTheme.typography.bodyLarge) {
+                Text(text = stringResource(id = R.string.file_upload_take_photo))
+              }
+              Icon(imageVector = HedvigIcons.Camera, contentDescription = "photo icon")
+            }
+          }
+          Spacer(modifier = Modifier.height(4.dp))
+          TextButton(
+            onClick = {
+              filePicker.launch("*/*")
+              showFileTypeSelectBottomSheet = false
+            },
+            modifier = modifier
+              .fillMaxWidth()
+              .padding(horizontal = 16.dp),
+            shape = MaterialTheme.shapes.squircleMedium,
+          ) {
```

Comment:
How do we want the ripple to play out here?
This view kinda looks like to me that clicking it would get the ripple to go all the way to the edges like in a typical list.
However if we want it to look/feel like a button which does not go all the way to the edges, maybe we can reuse the design that the bottom sheet in the home screen has? Where the buttons there look more like cards, with a slightly different background? What do you think?